src/ludamus/client/src/brand-mark.css

```
@@ -0,0 +1,82 @@
+/* Logo builder lockup: icon scaled per rule, wordmark in currentColor */
+.logo-lockup .lockup-icon {
+  transform: translate(10.86px, 22.17px) scale(0.3191);
+  transition: transform 200ms var(--ease-out);
+}
+
+.logo-lockup.chunky .lockup-icon {
+  transform: translate(9.96px, 21.27px) scale(0.3341);
+}
+
+.logo-lockup.mono .bm-t {
+  fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)